src/app/components/Checkbox.tsx

```diff
@@ -40,11 +41,22 @@ const CustomCheckbox = styled(
 `;
 
 // tslint:disable-next-line:variable-name
-const Checkbox = ({children, className, ...props}: React.PropsWithChildren<Props>) => <label className={className}>
+const Label = styled.label`
+  ${(props) => props.disabled ? `
```

Comment:
its a little confusing to me that these styles aren't with the other checkbox styles below